web_api/pyproject.toml

```diff
@@ -41,6 +41,9 @@ types-redis = "^0.1.10"
 types-dj-database-url = "^0.5.0"
 responses = "^0.21.0"
 pytest-django = "^4.5.2"
+types-stripe = "^2.59.10"
+pydantic = "^1.9.1"
```

Comment:
updating due to mypy issue: https://github.com/samuelcolvin/pydantic/issues/3528